target/linux/ar71xx/files/arch/mips/ath79/mach-gl-ar750s.c

```diff
@@ -189,5 +189,5 @@ static void __init  gl_ar750s_setup(void)
 }
 
 
-MIPS_MACHINE(ATH79_MACH_GL_AR750S, "GL-AR750S", "GL-AR750S",
+MIPS_MACHINE(ATH79_MACH_GL_AR750S, "GL-AR750S", "GL.iNet GL-AR750S",
```

Comment:
This change is cosmetic. Since ar71xx is only about backporting, I'd remove this change.

Reply:
I changed the name to what it's called in ath79. Changing the name for ar71xx might have no effect in OpenWrt, but it would make the ar71xx --> ath79 transition easier for other projects (e.g. gluon). We already had a bug in gluon where the device autoupdate stopped working because other GL.iNet devices got renamed and nobody noticed (https://github.com/freifunk-gluon/gluon/pull/1640). Therefore I'd like to keep this change.

Reply:
1. In any case, it's a separate change. So it should go into a separate commit.
2. As you nicely depicted, changing names will cause issues downstream. That's why backporting cosmetic changes to a stable target is typically not done. With the transition from ar71xx to ath79, you will have so many cases of changed names, additional images, ambiguous board names etc. that this single case will just be one of many. And I just do not see why a rename now will be better than a rename when you change to ath79.

Anyway, maybe @blocktrron has a different view on this subject. I'd not touch ar71xx for cosmetic reasons anymore. If you put the diag.sh change into a separate commit and extend the commit message as requested, I'd merge this part relatively quickly.

Reply:
I see. I'll remove the name change for this PR and open a second one for that if necessary.